packages/@vuepress/theme-default/src/client/components/SidebarItems.vue

```diff
@@ -1,8 +1,47 @@
 <script setup lang="ts">
 import SidebarItem from '@theme/SidebarItem.vue'
+import { watch } from 'vue'
+import { useRouter } from 'vue-router'
 import { useSidebarItems } from '../composables'
 
 const sidebarItems = useSidebarItems()
+
+const route = useRouter()
```

Comment:
The variable name `foo` should be consistent with `useFoo`. i.e. `const router = useRouter`

Reply:
I have changed it.